pages/api/issue/[...ids].ts

```diff
@@ -44,16 +53,82 @@ async function get(req: NextApiRequest, res: NextApiResponse) {
   return res.status(200).json(issue);
 }
 
-export default LogAccess(WithCors(async function GetIssues(req: NextApiRequest,
-                                        res: NextApiResponse) {
+async function put(req: NextApiRequest, res: NextApiResponse) {
+  const {
+    ids: [repoId, ghId, networkName],
+  } = req.query;
+
+  const {
+    body,
+    tags
+  } = req.body;
+
+  const issueId = [repoId, ghId].join("/");
+
+  const network = await models.network.findOne({
+    where: {
+      name: {
+        [Op.iLike]: String(networkName).replaceAll(" ", "-")
+      }
+    }
+  });
+
+  if (!network) return res.status(404).json({message:"Invalid network"});
+
+  const issue = await models.issue.findOne({
+    where: {
+      issueId,
+      network_id: network?.id
+    },
+    include: [{ association: "repository" }]
+  });
+
+  if (!issue) return res.status(404).json({message: "bounty not found"});
+
+  if(issue.state === 'draft'){
```

Comment:
missing logic for if user making PUT actually is owner of the bounty

Reply:
the logic is in  `middleware\issue-route.ts`